package.json

```diff
@@ -8,7 +8,7 @@
    "scripts": {
      "build": "babel src -d lib"
```

Comment:
This script needs to be updated with:
```
browserify matrix-vis.js -t babelify --outfile bundle.js
```

once you have all the dependencies. You still need `babelify`.